api/src/test/java/uk/gov/hmcts/bar/api/unittests/PaymentInstructionControllerTest.java

```diff
@@ -0,0 +1,362 @@
+package uk.gov.hmcts.bar.api.unittests;
+
```

Comment:
@Jennings-HMCTS - Are we not deleting the old test classes for which you have written a new test classes?

Reply:
@aravindan89 The Old test class is the integration test while this is the Unit test class and hence they should be retained